src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 4;
+    private static final int MAX_NUMBER = 20;
```

Comment:
MAX_NUMBER of what?

Reply:
what about this comment?  MAX_NUMBER - not informative constant name
```suggestion
    private static final int MAX_FIGURE_PARAMETER_NUMBER = 20;
```